server/src/main/java/org/elasticsearch/common/Strings.java

```diff
@@ -137,7 +137,13 @@ public static boolean hasText(CharSequence str) {
         if (hasLength(str) == false) {
             return false;
         }
-        return str.chars().anyMatch(c -> Character.isWhitespace(c) == false);
+        int strLen = str.length();
```

Comment:
This is probably not hot much after this change, but I still find it weird to have a utility like this that is "slow" like this.